R/convenience-functions.R

```diff
@@ -48,7 +58,7 @@
 #' Sebastian Funk
 #' medRxiv 2023.01.23.23284722
 #' \doi{https://doi.org/10.1101/2023.01.23.23284722}
-#' <https://www.medrxiv.org/content/10.1101/2023.01.23.23284722v1> # nolint
+#' <https://www.medrxiv.org/content/10.1101/2023.01.23.23284722v1> 
```

Comment:
the CI linter gives a warning here without #nolint. But ah well, it gives warnings everyhwere 👀